config/initializers/cors.rb

```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
+
+# Avoid CORS issues when API is called from the frontend app.
+# Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin AJAX requests.
+
+# Read more: https://github.com/cyu/rack-cors
+
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins "*"
+
+    resource "*",
+      headers: :any,
+      methods: [:get, :post, :put, :patch, :delete, :options, :head]
+  end
```

Comment:
- [ ] Let's fix the Cors issue that is preventing the messages from reaching the front-end app.


![cors](https://github.com/ShaliniNayan/hello-rails-back-end/assets/53124548/1e57ed37-968e-434c-acf4-7dbf2b7776d6)

Reply:
![image](https://github.com/ShaliniNayan/hello-rails-back-end/assets/117961613/acfcb2d7-9703-4bd2-b0b8-42f06b9fc9f0)

Reply:
![image](https://github.com/ShaliniNayan/hello-rails-back-end/assets/117961613/61d7ead8-7186-445e-aab2-739ee0a2009c)